cfbs/masterfiles/analyze.py

```diff
@@ -0,0 +1,73 @@
+# TODO merge this with ENT-12099 branch cfbs analyze.py
+import os
+
+from cfbs.utils import dict_sorted_by_key, file_sha256
+
+IGNORED_PATH_COMPONENTS = [".git/", ".gitignore", ".gitattributes"]
+# ignore a path iff it contains a component (single file or directory) from this list
```

Comment:
```suggestion
# ignore a path if it contains a component (single file or directory) from this list
```

Reply:
This one is on purpose, "iff" means "if and only if".

Reply:
Reworded in https://github.com/cfengine/cfbs/pull/208/commits/b516b19c038d149e10df342e60b71cb777cc34b5